src/ol/format/gpxformat.js

```diff
@@ -97,7 +97,7 @@ ol.format.GPX.parseLink_ = function(node, objectStack) {
   goog.asserts.assert(node.localName == 'link', 'localName should be link');
   var values = /** @type {Object} */ (objectStack[objectStack.length - 1]);
   var href = node.getAttribute('href');
-  if (!goog.isNull(href)) {
+  if (href !== null) {
```

Comment:
Trying to imagine where someone would actually want `''` to make it through here.  Can't imagine it, but I think the conservative approach is ok too.